packages/webview_flutter/tizen/src/webview.cc

```diff
@@ -900,3 +910,34 @@ void WebView::HandleCookieMethodCall(
     result->NotImplemented();
   }
 }
+
+FlutterDesktopGpuBuffer* WebView::CopyGpuBuffer(size_t width, size_t height) {
+  if (!candidate_surface_) {
+    return nullptr;
+  }
+
+  std::lock_guard<std::mutex> lock(mutex_);
+  if (rendered_surface_) {
+    tbm_surface_destroy(rendered_surface_);
+    rendered_surface_ = nullptr;
+  }
+
+  rendered_surface_ = candidate_surface_;
+  candidate_surface_ = nullptr;
+
+  FlutterDesktopGpuBuffer* gpuBuffer = new FlutterDesktopGpuBuffer();
+  gpuBuffer->buffer = rendered_surface_;
+  gpuBuffer->width = width;
+  gpuBuffer->height = height;
+  return gpuBuffer;
+}
+
+void WebView::Destruction(void* buffer) {
```

Comment:
To make the name more descriptive,

```suggestion
void WebView::DestructBuffer(void* buffer) {
```

Reply:
:ok_hand: